src/tests/ilverify/TestDataLoader.cs

```diff
@@ -279,10 +279,10 @@ public TestResolver(Dictionary<string, string> simpleNameToPathMap)
                 _simpleNameToPathMap = simpleNameToPathMap;
             }
 
-            PEReader IResolver.ResolveAssembly(AssemblyName assemblyName)
+            PEReader IResolver.ResolveAssembly(AssemblyNameInfo assemblyName)
```

Comment:
This is fixing build break in ILVerify tests: https://dnceng-public.visualstudio.com/public/_build/results?buildId=672464&view=logs&j=3725811e-4a8b-5f6a-b03b-a353ee894148&t=d59fccfb-2b13-5841-75bf-bfec705ee16a

Reply:
Wondering why this was not caught in CI ran during PR. Is the difference just `Debug` vs. `Release`?

```
fail: /__w/1/s/build.sh -ci -arch arm64 -os linux -cross -s clr -c Debug 
pass: /__w/1/s/build.sh -ci -arch arm64 -os linux -cross -s clr+libs+host+packs -rc Release -c Release -lc Release   
```

Reply:
Right, the default CI mix builds checked and release coreclr builds only. 99.9+% of debug build breaks are caught by checked builds. It is only outliers like this one (that looks like a C++ compiler bug to me) that can sneak through.